go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/samalba/dockerclient v0.0.0-20160414174713-91d7393ff859 // indirect
 	github.com/shirou/gopsutil/v3 v3.23.2
-	github.com/spf13/cobra v1.6.1
+	github.com/spf13/cobra v1.6.1-0.20230322224151-4dd4b25de384
```

Comment:
They released v1.7.0 which includes the change you mentioned, please update to that instead

Reply:
Yes, I will do it, they made a release when I pulled my changes 😁